lib/testers/rule-tester.js

```diff
@@ -69,8 +69,8 @@ const RuleTesterParameters = [
     "code",
     "filename",
     "options",
-    "args",
-    "errors"
+    "errors",
```

Comment:
I can do it, but as you said, it seems unrelated to this issue, so maybe it should go as a different PR.